chart/values.yaml

```diff
@@ -0,0 +1,49 @@
+CliVersion: linkerd/cli edge-19.2.5
```

Comment:
It would be nice to group and comment on these. I don't think it is a requirement for this PR as I can imagine the `install` code would change. That said, I'd love to see something like:

```yaml
images:
    proxy: gcr.io/linkerd-io/proxy:edge-19.2.5
    controller: gcr.io/linkerd-io/controller:edge-19.2.5

tls:
    certFileName: certificate.crt

proxy:
    autoInjectEnabled: false
    containerName: linkerd-proxy
```

What do you think @cohix create a separate issue for that?

Reply:
Is it possible to have a CLI command that dumps these? I'm thinking something similar to `linkerd doc`? Again, not something required for this PR to land, but maybe a follow up issue?

Reply:
Something like `helm inspect values`?

Reply:
@ihcsim other way around I think. I'd like to have CI auto-update `chart/values.yaml` based off the config struct in the code itself.

Reply:
I re-organised the values.yaml to group them like you mentioned.